### DISCUSSION TITLE
qq怎么卸载补丁？

### DISCUSSION THREAD

**Discussion by 3c9VKAUjrXMzE4ss** [huiyadanli/RevokeMsgPatcher#764] [Q&A] [answered]
桌面版。之前好奇打完补丁，感觉用不上，基本不咋用qq，现在想卸载，没看到教程说啊？现在一开qq任务栏下面就有个qq标志的任务框，但是打不开，也没有标题名就是一个企鹅logo的任务标。应该是这个补丁的事情吧，强迫症想卸载了……有大佬救一下？

**Comment by huiyadanli** [comment, marked as answer]
首先你的问题和补丁无关。

其次，备份还原就可以卸载补丁。或者覆盖安装QQ。
